extensions/amp-consent/0.1/test/test-consent-info.js

```diff
@@ -150,6 +159,30 @@ describes.fakeWin('ConsentInfo', {}, () => {
       ).to.deep.equal(
         dict({
           'consentStringType': CONSENT_STRING_TYPE.US_PRIVACY_STRING,
+          'additionalConsent': undefined,
+        })
+      );
+      expect(
+        convertStorageMetadata({
+          undefined,
+          [METADATA_STORAGE_KEY.ADDITIONAL_CONSENT]: '1~1.35.41.101',
+        })
+      ).to.deep.equal(
+        dict({
+          'consentStringType': undefined,
+          'additionalConsent': '1~1.35.41.101',
+        })
+      );
+      expect(
+        convertStorageMetadata({
+          [METADATA_STORAGE_KEY.CONSENT_STRING_TYPE]:
+            CONSENT_STRING_TYPE.US_PRIVACY_STRING,
+          [METADATA_STORAGE_KEY.ADDITIONAL_CONSENT]: '1~1.35.41.101',
+        })
+      ).to.deep.equal(
+        dict({
+          'consentStringType': CONSENT_STRING_TYPE.US_PRIVACY_STRING,
+          'additionalConsent': '1~1.35.41.101',
```

Comment:
: ) when we add gdprApplies, I think it's sufficient to only test metadata with everything defined. Or there'll be 8 permutations : P